stylesheets/main.css

```diff
@@ -34,3 +34,30 @@ body {
   align-items: center;
   justify-content: center;
 }
+
+ul {
```

Comment:
You're selecting EVERY `ul` and `li` element. At the moment they're only being used in your navbar but you're preventing anyone else from using them! Please be more specific and use an id/class.